presto-main/src/test/java/com/facebook/presto/operator/project/TestPageProcessor.java

```diff
@@ -22,6 +22,9 @@
 import com.facebook.presto.spi.block.LazyBlock;
 import com.facebook.presto.spi.block.VariableWidthBlock;
 import com.facebook.presto.spi.type.Type;
+import com.facebook.presto.sql.gen.ExpressionProfiler;
```

Comment:
I'd like to see a test, where this effects the output scaling, and if possibly with and without the size scaling, so we can verify the integrated feature is working how we expected.